.github/workflows/release.yaml

```diff
@@ -0,0 +1,30 @@
+name: "Release"
+
+on: workflow_dispatch
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: create branch and tag
+        run: |
+          OLD_VERSION=$(git describe --abbrev=0 --tags)
+          IFS=. read -r major minor patch <<< "$OLD_VERSION"
+          RELEASE_BRANCH="release-$major.$((++minor))" 
+          NEW_VERSION="$major.$minor.0-rc1"
```

Comment:
Might be worth adding a param for the one time we need to increase major? If param is unset just do inference otherwise use the param

Reply:
Yes, I can add a boolean flag for that purpose

Reply:
Done